packages.json

```diff
@@ -194,7 +189,6 @@
                 "xrdb"
             ],
             "sericea": [
-                "clipman",
```

Comment:
This was added in #273 and it does fill a gap in the upstream Fedora Atomic Sway image. Unless this is provided upstream, I am not inclined to remove this.

Reply:
@bsherman upstream already provides wl-clipboard